packages/twenty-ui/src/theme/constants/FontDark.ts

```diff
@@ -10,7 +10,7 @@ export const FONT_DARK = {
     light: GRAY_SCALE.gray50,
     extraLight: GRAY_SCALE.gray55,
     inverted: GRAY_SCALE.gray100,
-    danger: COLOR.red,
+    danger: COLOR.red70,
```

Comment:
@Bonapara I feel this was right before @its-id change.
the danger color font should be red #f83e3e according to Figma
![Uploading image.png…]()

Could you confirm?

Reply:
So, to understand the fix: a brief about the issue itself. Earlier the FontLight and FontDark both were hardcoded to take only single color which turned out to be an issue in disabled mode. (ref to issue screenshot)

The fix revolved around the usage of FontDark and FontLight to work same as BorderLight and BorderDark in disabled mode. So, I made FontDark and FontLight danger color to use same value same as being used by BorderLight (#fed8d8) and BorderDark (#441f1f). This now becomes much more cleaner and easier to use as acc. to current changes, fontDark and fontLight danger is being used only for disabled mode. 

And as for the other files which were previously using font.color.danger value  which was basically COLOR.red (#f83e3e), I switched to theme.color.red making sure their color value doesn't change.